test/unit/test-utils.js

```diff
@@ -890,4 +906,51 @@ describe('Lambda Utils', () => {
 
     });
 
+    describe('invokeLambda', () => {
+        const alias = 'aliasName';
+        const arn = 'arn:aws:lambda:eu-west-1:XXX:function:name';
+        const payload = {testKey: 'test-value'};
+
+        let consoleLogStub;
+
+        const invokeLambdaAndAssertOnConsoleLog = async({disablePayloadLogs, isPayloadInConsoleLog}) => {
+            utils.invokeLambda(arn, alias, payload, disablePayloadLogs);
+
+            const consoleLogArg = consoleLogStub.firstCall.args[0];
+
+            expect(consoleLogArg).to.contain('Invoking function');
+            expect(consoleLogArg.includes('with payload')).to.be(isPayloadInConsoleLog);
+        };
+
+        before(() => {
+            if (consoleLogSetupStub) {
+                consoleLogStub = consoleLogSetupStub;
+            } else {
+                consoleLogStub = sinon.stub(console, 'log');
```

Comment:
Why is this needed here?

Reply:
This is a workaround because of how we [setup tests](https://github.com/alexcasalboni/aws-lambda-power-tuning/blob/master/test/setup.spec.js#L9) with sinon stubs.

I wanted to keep this setup where we could choose to comment out the setup stubs if desired, without breaking this specific test logic.

Open to suggestions if we want to revisit how tests are set up now, or if we want to improve how we handle stubs in this specific test case.

Reply:
Got it! I think that's fine for now :)